app/assets/javascripts/components/common/academic_system.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const createReactClass = require('create-react-class');
+
+const AcademicSystem = createReactClass({
+
+  getInitialState: function () {
+    return {
+      selectedOption: 'Semester'
+    };
+  },
+
+  handleChange: function () {
+    this.props.onChange(this.props.myValue);
+  },
+
+  render: function () {
+    return (
+      <div>
+        <label>
+          <input type = "radio" name="academic_system" defaultChecked={this.state.selectedOption === 'Semester'} onChange={this.handleOptionChange}/>
+          Semester
```

Comment:
This should probably have an `Other` option as well. I suggest putting the options in an array, and generating the inputs dynamically, so that it will be easy to edit the options later.